src/app/frontend/deploy/deploy.scss

```diff
@@ -18,3 +18,38 @@
   // TODO(bryk): Find a better, application wide constant for the min width.
   min-width: 600px;
 }
+
+.kd-deploy-label-list {
+  > md-list-item {
+    min-height: 10px;
+    > kd-label {
+      padding-bottom: 0;
+      > ng-form {
+        padding-bottom: 0;
+        > md-input-container {
+          padding-bottom: 0;
+        }
+      }
+    }
+  }
+}
+
+.kd-deploy-labels-title {
+  padding-left: 5px;
+}
+
+.kd-deploy-labels-header {
+  font-size: 0.85em;
+  padding: 0 10px !important;
```

Comment:
`!important`? It should be almost always avoided, unless you're writing framework-y stuff. Rely on rule specificity or order. Otherwise, we'll, sooner or later, run into problems caused by this. This always happens in web dev.